common/flatpak-dir.c

```diff
@@ -14068,27 +14088,24 @@ flatpak_dir_get_default_locale_languages (FlatpakDir *self)
   if (accounts_proxy != NULL)
     get_locale_langs_from_accounts_dbus (accounts_proxy, langs);
 
-  /* If none were found, fall back to using all languages */
-  if (langs->len == 0)
-    return g_new0 (char *, 1);
-
   g_ptr_array_sort (langs, flatpak_strcmp0_ptr);
   g_ptr_array_add (langs, NULL);
 
-  return (char **) g_ptr_array_free (g_steal_pointer (&langs), FALSE);
+  return flatpak_strv_merge (extra_languages, (char **) langs->pdata);
```

Comment:
All the other codepaths sort the result, but this merge doesn't.

Reply:
Oh yeah! Sorry - I have deleted the previous sort for langs and added `sort_strv` for the return on `flatpak_strv_merge`